resources/production/config-example.toml

```diff
@@ -41,6 +44,27 @@ pending_vertex_timeout = '30min'
 # If it is more than that, consensus will pause, and resume once the executor has caught up.
 max_execution_delay = 3
 
+[consensus.highway.round_success_meter]
+# The number of most recent rounds we will be keeping track of.
+num_rounds_to_consider = 40
+
+# The number of successful rounds that triggers us to slow down: With this many or fewer
+# successes per `num_rounds_to_consider`, we increase our round exponent.
+num_rounds_slowdown = 10
+
+# The number of successful rounds that triggers us to speed up: With this many or more successes
+# per `NUM_ROUNDS_TO_CONSIDER`, we decrease our round exponent.
```

Comment:
…and these

Reply:
Done in [`2c5d549` (#1128)](https://github.com/CasperLabs/casper-node/pull/1128/commits/2c5d5495577dc044908a7f25024804ce54dbacb2)